src/KnawatMP.js

```diff
@@ -150,7 +150,15 @@ class KnawatMP {
    */
   async __fetch(method, path, options = {}) {
     await this.setAuthHeaders(options.auth);
-    let requestUrl = `${KnawatMP.baseUrl}${path}`;
+    let asyncPath = '';
+    if (method.toLocaleLowerCase() !== 'get' && !path.split('/').includes('token')) {
+      asyncPath =
+        Number(process.env.SET_ASYNC) === 0 || Number(options.SET_ASYNC) === 0
```

Comment:
This will resolve to `false` or `'/async'`
```suggestion
        Number(options.SET_ASYNC) === 0
```

Reply:
updated the code